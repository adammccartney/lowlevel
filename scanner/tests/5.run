./scanner tests/5.in
