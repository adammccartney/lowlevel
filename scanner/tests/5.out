50000
61000
